XLS-0070d-credentials/README.md

```diff
@@ -0,0 +1,595 @@
+2. **VC Access:** The DID document provides a link to the relevant Verifiable Credentials.
+3. **VC Verification:** Based on access permissions, verifiers can then view and verify the retrieved VCs.
+
+However, this proposal introduces an optional enhancement. Within the `Credential` object, the `URI` field can also point to the VC. It's important to note that this pointer is entirely optional, and the core verification process using DID resolution will still be the primary method for XRPL credential verification.
+
+It is recommended that anyone who holds a credential also have a DID object, to increase compliance with the W3C VC spec.
+
+Note that the issuer of the on-chain `Credential` does not have to be the same as the issuer of the VC, and the on-chain object can instead essentially serve as an on-chain attestation from the issuer for the VC.
+
+## 11. Invariants
+
+### 11.1. Reserves
+
+The burden of reserve should be with the issuer, and the credential should be in the issuer's owner directory, if the `lsfAccepted` flag is off. The burden of reserve should be with the subject and the credential should be in the subject's owner directory, if the `lsfAccepted` flag is on.
+
+### 11.2. The `DepositPreauth` Object
+
+A `DepositPreauth` ledger object always has exactly one of the `Authorize` field and the `AuthorizeCredentials` field.
+
+If the `AuthorizeCredentials` field is included, the array contains between 1 and 8 credentials.
+
+## 12. Examples
+
+In this example, a trusted issuer, Isabel, is issuing a credential that indicates that Alice is KYC'd. Verity is setting up her account to only interact with accounts that Isabel has attested to being properly KYC'd.
+
+For ease of reading, some of the common transaction fields, such as signatures and public keys, have been left out of this example.
+
+### 12.1. `CredentialCreate`
+
+Isabel creates the credential for Alice, after confirming her KYC status off-chain.
+
+```typescript
+{
+    TransactionType: "CredentialCreate",
+    Account: "rISABEL......",
+    Subject: "rALICE.......",
+    CredentialType: "4B5943", // "KYC" in hex
+    Expiration: 789004799, // the end of the year
+    URI: "isabel.com/credentials/kyc/alice" // This will be converted into hex
+}
+```
+
+### 12.2. The `Credential` Object
+
+This is the object created by the `CredentialCreate` transaction.
+
+```typescript
+{
+    LedgerEntryType: "Credential",
+    Flags: 0,
+    Subject: "rALICE.......",
+    Issuer: "rISABEL......",
+    CredentialType: "123ABC",
+    Expiration: 789004799, // the end of the year
+    URI: "isabel.com/credentials/kyc/alice" // This will be converted into hex
+}
+```
+
+### 12.3. `CredentialAccept`
+
+Alice accepts the credential, thereby making it valid.
+
+```typescript
+{
```

Comment:
it would be instructive to include the updated `Flags` field in the Credential ledger object.